spec/lexers/solidity_spec.rb

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*- #
+
+describe Rouge::Lexers::Solidity do
+  let(:subject) { Rouge::Lexers::Solidity.new }
+
+  describe 'guessing' do
+    include Support::Guessing
+
+    it 'guesses by filename' do
+      assert_guess :filename => 'foo.sol'
+      assert_guess :filename => 'foo.solidity'
+    end
+
+    it 'guesses by mimetype' do
+      assert_guess :mimetype => 'text/x-solidity'
+    end
+
+    # it 'guesses by source' do
+    #   assert_guess :source => 'pragma solidity'
```

Comment:
This block is commented, since this line failed to pass tests for me (when run locally with `bundle exec rake`): the guesser hit `PlainText` instead of `Solidity` - perhaps on the "reverse" pass?..

Reply:
"Fixed" by using `starts_with?` in the lexer's `detect()`, instead of `shebang?`. :drum: